src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporter.cs

```diff
@@ -63,7 +64,20 @@ public OtlpTraceExporter(OtlpExporterOptions options)
         internal OtlpTraceExporter(OtlpExporterOptions options, OtlpCollector.TraceService.ITraceServiceClient traceServiceClient = null)
         {
             this.options = options ?? throw new ArgumentNullException(nameof(options));
-            this.headers = options.Headers ?? throw new ArgumentException("Headers were not provided on options.", nameof(options));
+            this.headers = GetMetadataFromHeaders(options.Headers);
+
+            if (options.TimeOut.HasValue)
+            {
+                if (options.TimeOut.Value <= 0)
+                {
+                    throw new ArgumentException("TimeOut vaue provided is not a positive numbe", nameof(options.TimeOut));
```

Comment:
```suggestion
                    throw new ArgumentException("TimeOut vaue provided is not a positive number", nameof(options.TimeOut));
```

Reply:
Fixed the typos and moved this logic to Export method.